__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+const categoriesData = require('../db/data/test-data/categories')
+
+beforeEach(() => {
+    return seed(data);
+})
+
+describe('GET Categories', () => {
+    test('should return object with one entry: {"categories" : <all categories data>}', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
+        .then((res) => {
+            const categories = res.body.categories;
+
+            console.log(categories);
+
+            expect(Array.isArray(categories)).toBe(true);
```

Comment:
You can remove this assertion as we implicitly check that with have an array with the forEach later on